.circleci/config.yml

```diff
@@ -28,8 +28,17 @@ templates:
       default: ""
       type: string
 
+  default-environment: &default-environment
+    GETH_URL_LINUX: 'https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.8.20-24d727b6.tar.gz'
+    GETH_URL_MACOS: 'https://gethstore.blob.core.windows.net/builds/geth-darwin-amd64-1.8.20-24d727b6.tar.gz'
+    GETH_VERSION: '1.8.20'
+    SOLC_URL_LINUX: 'https://github.com/ethereum/solidity/releases/download/v0.4.23/solc-static-linux'
+    SOLC_URL_MACOS: 'https://www.dropbox.com/s/4amq3on2ds1dq36/solc_0.4.23?dl=0'
```

Comment:
Is this an official link for Solc MacOS?

Reply:
No, compiled by me ;)

Reply:
No, this is a binary one of us (in this case @palango IIRC) compiled and uploaded. 
So far there are unfortunately [no official solidity macOS binaries](https://github.com/ethereum/solidity/issues/3168)

Although now with circle we could probably switch back to compiling it ourselves and storing the result in the cache.